packages/@adobe/spectrum-css-temp/components/popover/index.css

```diff
@@ -80,7 +80,7 @@ governing permissions and limitations under the License.
     @inherit: %spectrum-overlay--left--open;
   }
   .spectrum-Popover-tip {
-    right: -16px;
+    right: calc(-6px - var(--spectrum-global-dimension-size-125));
```

Comment:
where'd you get this from?

Reply:
So from testing the `right` here needs to be `-16px` for medium and `-19px` for large but unfortunately there isn't a global size for the 16px/19px pairing so I did this calculation.

Reply:
In hindsight, I probably should've made 2 new variables in spectrum-medium and spectrum-large, will do that